client/index.html

```diff
@@ -16,6 +16,7 @@
         <!-- bower components -->
         <script src="bower_components/angular/angular.min.js"></script>
         <script src="bower_components/angular-ui-router/release/angular-ui-router.min.js"></script>
+        <script src="bower_components/angular-messages/angular-messages.min.js"></script>

```

Comment:
its a script so I dont think there is a need to add it to index.gulp.html, I add the script source to the gulpfile so it will concat that into the 1 js file it puts in index_gulp.html